ee/modules/110-istio/openapi/config-values.yaml

```diff
@@ -61,6 +61,17 @@ properties:
               - `LoadBalancer` — is a recommended method if you have a cloud-based cluster and it supports Load Balancing.
               - `NodePort` — for installations that do not have the LB.
             default: LoadBalancer
+          nodePort:
+            type: object
+            description: Special settings for NodePort inlet.
+            default: {}
```

Comment:
Can we remove default for nodePort and add `port` field into required section? I think it'll look better.

Reply:
We can use `oneOf` for requiring field based on enum value.
For example, we use this technique in cluster configuration for choice required fields between static and cloud params. See it [here](https://github.com/deckhouse/deckhouse/blob/6822fac856e94ff500878d120b2362ccef70b047/candi/openapi/cluster_configuration.yaml#L84)

Could we use it here for requiring nodePort object if `inlet` is `NodePort`?

Additional link https://stackoverflow.com/a/38781027

Reply:
We decided to keep the original behavior.